src/test/java/uk/gov/hmcts/juror/api/moj/report/grouped/PersonAttendingDetailReportTest.java

```diff
@@ -0,0 +1,178 @@
+    }
+
+    @BeforeEach
+    @Override
+    public void beforeEach() {
+        this.courtLocationRepository = mock(CourtLocationRepository.class);
+        super.beforeEach();
+    }
+
+    @Override
+    public PersonAttendingDetailReport createReport(PoolRequestRepository poolRequestRepository) {
+        return new PersonAttendingDetailReport(poolRequestRepository, this.courtLocationRepository);
+    }
+
+    @Override
+    protected StandardReportRequest getValidRequest() {
+        return StandardReportRequest.builder()
+            .reportType(report.getName())
+            .date(LocalDate.now())
+            .includeSummoned(false)
+            .build();
+    }
+
+    @Override
+    public void positivePreProcessQueryTypical(JPAQuery<Tuple> query, StandardReportRequest request) {
+        String locCode = "415";
+        TestUtils.mockSecurityUtil(BureauJwtPayload.builder().locCode(locCode).userType(UserType.COURT).build());
+
+
+        StandardReportRequest requestMock = mock(StandardReportRequest.class);
+        when(requestMock.getDate()).thenReturn(LocalDate.now());
+
+        report.preProcessQuery(query, request);
+        verify(query, times(1))
+            .where(QJurorPool.jurorPool.nextDate.eq(requestMock.getDate()));
+        verify(query, times(1))
+            .where(QJurorPool.jurorPool.pool.courtLocation.locCode.eq(locCode));
+        verify(query, times(1))
+            .where(QJurorPool.jurorPool.status.status.in(IJurorStatus.RESPONDED,
+                                                         IJurorStatus.PANEL,
+                                                         IJurorStatus.JUROR));
+    }
+
+
+    @Test
+    void positivePreProcessQueryWithSummoned() {
+        String locCode = "415";
+        TestUtils.mockSecurityUtil(BureauJwtPayload.builder().locCode(locCode).userType(UserType.COURT).build());
+
+        StandardReportRequest requestMock = mock(StandardReportRequest.class);
+        when(requestMock.getDate()).thenReturn(LocalDate.now());
+
+        JPAQuery<Tuple> query = mock(JPAQuery.class,
+                                     withSettings().defaultAnswer(RETURNS_SELF));
+        StandardReportRequest request = getValidRequest();
+
+        request.setIncludeSummoned(true);
+
+        report.preProcessQuery(query, request);
+        verify(query, times(1))
+            .where(QJurorPool.jurorPool.nextDate.eq(requestMock.getDate()));
+        verify(query, times(1))
+            .where(QJurorPool.jurorPool.pool.courtLocation.locCode.eq(locCode));
+        verify(query, times(1))
+            .where(QJurorPool.jurorPool.status.status.in(IJurorStatus.SUMMONED,
+                                                         IJurorStatus.RESPONDED,
+                                                         IJurorStatus.PANEL,
+                                                         IJurorStatus.JUROR));
+
+        verifyNoMoreInteractions(query);
+
+        TestUtils.afterAll();
```

Comment:
This should be in a @AfterAll at the top of the test class instead of within the test (As you also setup security mocks in the other test)